src/Symfony/Component/Translation/Tests/Extractor/PhpExtractorTest.php

```diff
@@ -68,11 +68,15 @@ public function testExtraction($resource)
         $actualCatalogue = $catalogue->all();
 
         $this->assertEquals($expectedCatalogue, $actualCatalogue);
+
+        $filename = __DIR__.'/../fixtures/extractor/translation.html.php';
+        $this->assertEquals(['sources' => [$filename.':2']], $catalogue->getMetadata('single-quoted key'));
+        $this->assertEquals(['sources' => [$filename.':43']], $catalogue->getMetadata('other-domain-test-no-params-short-array', 'not_messages'));
     }
 
     public function resourcesProvider()
     {
-        $directory = __DIR__.'/../fixtures/extractor/';
+        $directory = __DIR__.'/../fixtures/extractor';
```

Comment:
I would revert this change as well to ease merging older branches to newer ones.

Reply:
This change is needed due tests failing on Travis with `PHP 7.3` and `deps=low`: https://travis-ci.org/symfony/symfony/builds/524492376